src/components/tables/RowActions/RowSelector.tsx

```diff
@@ -75,6 +76,12 @@ function RowSelector({
                     selectableTableStoreName={selectableTableStoreName}
                 />
             ) : null}
+
+            {selectableTableStoreName === SelectTableStoreNames.COLLECTION ? (
```

Comment:
Can you mark with a `todo` as this can be made a property so all settings are controlled directly by the parent.

I don't think we need to change now and can just handle it later.... mainly because I really want us to move more of the drilled prop settings we have into a context provider for the tables.